js/index.js

```diff
@@ -0,0 +1,56 @@
+"use strict";
+
+var guardianParser = new JsonParser();
+
+var executeAfterFetchingData = function () {
+  var titles = guardianParser.getTitles();
+  var urls = guardianParser.getUrls();
+  var bodies = guardianParser.getBodies();
+  var thumbnails = guardianParser.getThumbnails();
+  
+  var articles = new ArticleList();
+  var article;
+  
+  for (var i = 0; i < titles.length; i++) {
```

Comment:
I've commented on this elsewhere, but it feels like this code could be removed completed and the articles generated by the parser